components/MarqueeText/samples/MarqueeText.md

```diff
@@ -0,0 +1,24 @@
+---
+title: MarqueeText
+author: Avid29
+description: A control for scrolling text in a marquee fashion.
+keywords: Marquee, Control, Text
+dev_langs:
+  - csharp
+category: Controls
+subcategory: StatusAndInfo
+discussion-id: 231 
+issue-id: 0
+---
+
+# MarqueeText
+
+For more information about this experiment see:
+
+- Discussion: https://github.com/CommunityToolkit/Labs-Windows/discussions/231
+
+- Issue: TODO: PASTE LINK HERE
+
+The MarqueeText control allows text to scroll in a marquee fashion. The control is heavily templated and many changes can be made by modifying the style.
```

Comment:
Maybe mention the Behavior / Direction / Speed / RepeatBehavior here?

Reply:
Yeah, I kind of ignored the markdown file. I just pushed a new commit with some more full docs, but I couldn't get multiple demo pages without the compiler blowing up 🤷‍♂️

Reply:
Changes looking great :)! Thanks!